src/encoded/tests/features/conftest.py

```diff
@@ -46,12 +49,14 @@ def index_workbook(request, app):
         'REMOTE_USER': 'TEST',
     }
     testapp = TestApp(app, environ)
-
-    from encoded.loadxl import load_all
-    from pkg_resources import resource_filename
-    inserts = resource_filename('encoded', 'tests/data/inserts/')
-    docsdir = [resource_filename('encoded', 'tests/data/documents/')]
-    load_all(testapp, inserts, docsdir, log_level=log_level)
+    global index_workbook_loaded
```

Comment:
Cherry-picked from `load-indexing-once` branch. Probably should be merged separately.